index.html

```diff
@@ -1 +1 @@
-<h1>Hello, World!</h1>
+<h1>Hello, World!meow</h1>
```

Comment:
You forgot capital letter M

Reply:
ah sorry bro, ill do that later. This is a lot to take in